docs/pages/management/export-audit-events/panther.mdx

```diff
@@ -0,0 +1,226 @@
+from Teleport's events API, and forwards them to Fluentd.
+
+(!docs/pages/includes/install-event-handler.mdx!)
+
+## Step 2/6. Generate a plugin configuration
+
+(!docs/pages/includes/configure-event-handler.mdx!)
+
+## Step 3/6. Create a user and role for reading audit events
+
+(!docs/pages/includes/plugins/event-handler-role-user.mdx!)
+
+## Step 4/6. Create teleport-event-handler credentials
+
+### Enable impersonation of the Event Handler user
+
+(!docs/pages/includes/plugins/rbac-impersonate-event-handler.mdx!)
+
+### Export an identity file for the Event Handler plugin user
+
+Give the plugin access to a Teleport identity file. We recommend using Machine
+ID for this in order to produce short-lived identity files that are less
+dangerous if exfiltrated, though in demo deployments, you can generate
+longer-lived identity files with `tctl`:
+
+<Tabs>
+<TabItem label="Machine ID">
+(!docs/pages/includes/plugins/tbot-identity.mdx secret="teleport-event-handler-identity"!)
+</TabItem>
+<TabItem label="Long-lived identity files">
+(!docs/pages/includes/plugins/identity-export.mdx user="teleport-event-handler" secret="teleport-event-handler-identity"!)
+</TabItem>
+</Tabs>
+
+## Step 5/6. Create a Dockerfile with Fluentd and the S3 plugin
+
+To send logs to Panther, you need to use the Fluentd output plugin for S3. Create
+a `Dockerfile` with the following content:
+
+```Dockerfile
+FROM fluent/fluentd:edge
+USER root
+RUN fluent-gem install fluent-plugin-s3
+USER fluent
+```
+
+Build the Docker image:
+```shell
+$ docker build -t fluentd-s3 .
+```
+
+<Admonition type="tip" title="Testing Locally?">
+
+If you're running Fluentd in a local Docker container for testing, you can adjust
+the entrypoint to an interactive shell as the root user, so you can install the
+plugin before starting Fluentd:
+
+```code
+$ docker run -u $(id -u root):$(id -g root) -p 8888:8888 -v $(pwd):/keys -v \
+$(pwd)/fluent.conf:/fluentd/etc/fluent.conf --entrypoint=/bin/sh -i --tty  fluentd-s3
+```
+
+</Admonition>
+
+### Configure Fluentd for Panter
+
+When you run the Teleport Event Handler, it will create a `fluent.conf` file.  This file needs to be updated
+to send logs to Panther. This means adding a `<filter>` and `<match>` section to the file. These sections
+will filter and format the logs before sending them to S3, The record_transformer is important to send the 
+right date and time format for Panther. 
+
+```xml
+# fluent.conf 
+# This is a sample configuration file for Fluentd to send logs to S3. 
+# Add the <filter> and <match> sections to the file.
+# <source>
+#     @type http
+#     port 8888
+#
+#     <transport tls>
+#         client_cert_auth true
+#         ca_path "/keys/ca.crt"
+#         cert_path "/keys/server.crt"
+#         private_key_path "/keys/server.key"
+#         private_key_passphrase "AUTOGENERATED"
+#     </transport>
+#
+#     <parse>
+#       @type json
+#       json_parser oj
+#
+#       # This time format is used by Go marshaller
```

Comment:
We might want to phrase this in terms of the Event Handler, since users may not make the connection immediately that it's a Go program.

Reply:
This is boiler plate but I removed it from the example.